examples/structured_data/classification_with_tfdf.py

```diff
@@ -146,23 +160,14 @@
 """
 
 
-def prepare_sample(features, target, weight):
-    for feature_name in features:
-        if feature_name in CATEGORICAL_FEATURES_WITH_VOCABULARY:
-            if features[feature_name].dtype != tf.dtypes.string:
-                # Convert categorical feature values to string.
-                features[feature_name] = tf.strings.as_string(features[feature_name])
-    return features, target, weight
-
-
 def run_experiment(model, train_data, test_data, num_epochs=1, batch_size=None):
```

Comment:
What about implementing the "Nit: What about having a **fit_args argument to the run_experiment function?" discussed before.

Reply:
Most of the Keras examples use the current `run_experiment` signature, and I didn't want to deviate from them.
Are you OK with that?

Reply:
I suspect that this is because those examples are training Neural networks (or another gradient descent based algorithm with mini-batching).